.circleci/config.yml

```diff
@@ -19,7 +19,9 @@ jobs:
 
       - run:
           name: Install app dependencies
-          command: pip install -r requirements.txt
+          command: |
+            pip install --upgrade pip
+            pip install -r requirements.txt --use-deprecated=legacy-resolver
```

Comment:
Based on the [build history](https://app.circleci.com/pipelines/github/openfisca/openfisca-doc), I fail to see how this issue manifests in CI. I see symptoms on Netlify, but no long builds on CircleCI.

Reply:
It didn't happen because cached pip version was too old - it was not using the new resolver. I could push an additional commit to expose the thing.

Reply:
Then why do we upgrade pip to its latest version in CI? 🤔 If there is a specific adjustment to make, why not limit the pip version until a patch lands?

Reply:
Sure, we could eventually cap the version to the one prior to the new resolver as well, both solutions accomplish IMHO the objective of letting the user know what's going on.

The reason why I did it like this was to have some sort of consistency with Netlify, where we don't have the option of specifying the version of pip to be used.

No preference.

Reply:
I did push a new commit and it [timed out](https://app.circleci.com/pipelines/github/openfisca/openfisca-doc/424/workflows/f4145b16-44e2-4d61-8e76-b5adb9f40615/jobs/1193) 😧